run-cluster-dask-jobs.sh

```diff
@@ -30,7 +31,7 @@ RUN_SCHEDULER=0
 # is typically run on 0 and putting the scheduler on 1 helps
 # distribute the load (I think, just based on getting OOM errors when
 # everything ran on 0).
-if [[ $SLURM_NODEID == 1 ]] || hasArg --scheduler-and-workers; then
+if [[ $SLURM_NODEID == 0 ]] || hasArg --scheduler-and-workers; then
```

Comment:
I believe we wanted to keep the Node with SLURM_NODEID=1 to start the scheduler to avoid OOM errors (assumption). Does this PR also fix that assumption ?

Reply:
No, but the script fails if we only have a single node run. I'll try to check if slurm has an environment variable that also exposes the number of nodes and use that information for the branching logic here.

Reply:
Oh, if we have a single node run it will setup the cluster differently as you can see in [run-nightly-benchmarks.sh](https://github.com/jnke2016/multi-gpu-tools/blob/main/run-nightly-benchmarks.sh#L101). it will use [LocalCudaCluster](https://github.com/rapidsai/cugraph/blob/branch-22.04/benchmarks/python_e2e/cugraph_dask_funcs.py#L172)  instead

Reply:
Unfortunately as you figured out someone outside of cugraph or running a different script will face this issue. perhaps inside this script we can check for the number of nodes and if it is one then use `$SLURM_NODEID == 0` ?

Reply:
Yeah this is what I wanted to add by check if slurm does expose some environment variable that mentions the number of nodes. Found it and added that logic